stable/nginx-ingress/templates/default-backend-deployment.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: extensions/v1beta1
```

Comment:
Could you make this, and the default-backend-svc.yaml optional the same way the lego stuff was?

This would allow a user to load in their own default server, like a regular nginx to serve out a site customized/branded 404 page.